src/utils.rs

```diff
@@ -109,15 +109,25 @@ pub fn walk_ptrs_ty<'t>(ty: ty::Ty<'t>) -> ty::Ty<'t> {
 ///         }
 ///     }
 #[macro_export]
-macro_rules! if_let_chain {
```

Comment:
Oh nevermind I see what you did here, this is awesome!

I still think the macro should be called `if_let_chain` though.